src/test/java/org/folio/permstest/RestVerticleTest.java

```diff
@@ -1043,7 +986,7 @@ public void testDeletePermsUsersPermissionsByIdAndPermissionnameMissingPermissio
   @Test
   public void testPostPermsPermissions(TestContext context) {
     String uuid = UUID.randomUUID().toString();
-    String permRequest = "{\"id\": \"" + uuid + "\", \"permissionName\":\"adname\",\"displayName\":\"addisplay\"}";
+    String permRequest = "{\"id\": \"" + uuid + "\", \"permissionName\":\"adname\",\"displayName\":\"addisplay\", \"mutable\":false}";
```

Comment:
Do not need this change. You new test below covers `PUT` with `mutable: false` case.

Reply:
yup, reverted.  I think this was an artifact of me changing my mind part way through implementing these tests.